src/qibolab/pulses.py

```diff
@@ -468,41 +468,43 @@ class SNZ(PulseShape):
     """
     Sudden variant Net Zero.
     https://arxiv.org/abs/2008.07411
+    (Supplementary materials)
```

Comment:
Should we reference the specific image as well ?

Reply:
Is it possible to reference an image in sphinx?

Reply:
I don't know